modules/server/src/network/aggregations/index.ts

```diff
@@ -1,2 +1,51 @@
+import { SupportedAggregation, SUPPORTED_AGGREGATIONS } from '../common';
+import { Aggregations, NetworkAggregation, NumericAggregations } from '../types';
+
 // TODO: implement
 export const resolveAggregations = (networkResults: void) => null;
```

Comment:
When you get around to working on this function I'd update this name. I'd add something descriptive to distinguish it's purpose from `resolveAggregations`